lib/db/query.js

```diff
@@ -59,7 +59,34 @@ async function getTests() {
   return result.rows;
 }
 
+async function getTestDB(testId) {
+  const query = `
+  SELECT 
+  t.name,
+  t.method,
+  t.url,
+  r.aws_name AS "region",
+  a.type,
+  a.property,
+  ar.actual_value,
+  ct.symbol AS "comparison_type",
+  a.expected_value,
+  ar.pass
+FROM tests t
+  JOIN test_runs tr ON tr.test_id = t.id
+  JOIN regions r ON r.id = tr.region_id
```

Comment:
Ah you did need the `region`, I figured Scott was the person to help, not me!! Good job guys!

Reply:
I cannot take credits for this query. It's all on Scott :D